templates/vhost/_file_header.epp

```diff
@@ -0,0 +1,43 @@
+# ************************************
+# Vhost template in module puppetlabs-apache
```

Comment:
can we follow right spacing for opening and closing if else blocks?
I don't see it consistent here?